### ISSUE TITLE
Apply inline attributes to closest block only

### ISSUE THREAD

**Issue by pirtleshell** [forestryio/markdown-it-kramdown-attrs#3]
# Problem
I am trying to render a line that looks like the following:

**Dinosaurs**: Hear them roar in a [new tab](link/to/dinosaurs/in/new/tab).

Where the link has the attribute `target="_blank"`. In HTML this looks like:

```html
<p><strong>Dinosaurs</strong>: Hear them roar in a <a href='link/to/dinosaurs/in/new/tab' target='_blank'>new tab</a>.</p>
```

Can something like this be written in markdown without writing the explicit HTML for the link?

---
# Possible solutions
I think the ideal way to write it in markdown is:

```markdown
**Dinosaurs**: Hear them roar in a [new tab](link/to/dinosaurs/in/new/tab){:target="_blank"}.
```

Currently, this is not recognized and is rendered as:
```html
<p><strong>Dinosaurs</strong>: Hear them roar in a <a href="link/to/dinosaurs/in/new/tab">new tab</a>{:target=&quot;_blank&quot;}.</p>
```

Separating out the lines, it is then recognized, but applies `target="_blank"` to the first token in the group instead of the link:
```markdown
**Dinosaurs**: Hear them roar in a
[new tab](link/to/dinosaurs/in/new/tab){:target="_blank"}
.
```
which renders as
```html
<p><strong target="_blank">Dinosaurs</strong>: Hear them roar in a
<a href="link/to/dinosaurs/in/new/tab">new tab</a>.</p>
```

It should be noted that the same behavior is exhibited by classes. Is there a way to apply the attribute only to the markdown block on which it appears?

**Comment by envygeeks** [comment]
Yes you are right, I plan on fixing this this week or next week and doing some general improvements.

**Comment by Neaox** [comment]
Is this still planning on being fixed?
